From 56c1261339dfc9519754b4af51829ba301c2fdb6 Mon Sep 17 00:00:00 2001
From: Paul Sajna <sajattack@postmarketos.org>
Date: Wed, 10 Sep 2025 19:47:20 -0700
Subject: [PATCH 126/212] MAINTAINERS: add Paul Sajna as maintainer for
 lg,sw49410-lh609qh1

Add myself (Paul Sajna) as maintainer for lg,sw49410-lh609qh1

Signed-off-by: Paul Sajna <sajattack@postmarketos.org>
---
 MAINTAINERS | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 71faf898fb44..586d3daf414e 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -8210,6 +8210,12 @@ T:	git https://gitlab.freedesktop.org/drm/misc/kernel.git
 F:	Documentation/devicetree/bindings/display/panel/lg,sw43408.yaml
 F:	drivers/gpu/drm/panel/panel-lg-sw43408.c
 
+DRM DRIVER FOR LG SW49410 PANELS
+M:	Paul Sajna <sajattack@postmarketos.org>
+S:	Maintained
+F:	Documentation/devicetree/bindings/display/panel/lg,sw49410.yaml
+F:	drivers/gpu/drm/panel/panel-lg-sw49410.c
+
 DRM DRIVER FOR LOGICVC DISPLAY CONTROLLER
 M:	Paul Kocialkowski <paulk@sys-base.io>
 S:	Supported
-- 
2.55.0

